spec/lib/contextual_logger_spec.rb

```diff
@@ -542,6 +542,7 @@ def inspect
 
     before(:each) do
       logger.register_secret(sensitive_data)
+      logger.register_secret_regex('(key|password|token|secret)[_a-z]*[\s\"]*(:|=>|=)[\s\"]*\K([0-9a-zA-Z_]*)')
```

Comment:
Please split these lines into their own contexts wrapped around each of the tests. As it is, because you're registering two ways, it's possible that either test is passing because of the other feature.